composer.json

```diff
@@ -28,7 +28,6 @@
         "drupal/search_api": "dev-1.x#ff41470b2b5399b83b86085d4b7a2c2f4e753370",
         "drupal/search_api_solr": "~1.0@beta",
         "drupal/state_machine": "~1.0-beta3",
-        "drupal/taxonomy_container": "~1.0",
```

Comment:
I would not lose time with creating patches upstream. Anyway, that selector I think will be changed because it doesn't allow to select the parent term and, as I recall, at some point Andrea said that parent terms selections should be allowed.